.cirrus.yml

```diff
@@ -6,7 +6,7 @@ env:
     #### Global variables used for all tasks
     ####
     # Name of the ultimate destination branch for this CI run
-    DEST_BRANCH: "main"
+    DEST_BRANCH: "release-5.26"
```

Comment:
@mtrmac shouldn't this be `release-5.27`?

Reply:
This may not be important, IIRC (though my memory is really fuzzy), we don't actually use this value anywhere in any CI scripts.

Reply:
> @mtrmac shouldn't this be `release-5.27`?

Totally. https://github.com/containers/image/pull/2084 

Thanks!

---

> we don't actually use this value anywhere in any CI scripts.

It’s used for the `git-validation` command run in `validate_task`. In _this_ case, where `release-5.27` is a continuation of `release-5.26`, it is just a small optimization; in the general case it’s a bit more complex (but it should only matter in rare cases, e.g. if `git-validation` rules were tightened.)